fe/fe-core/src/main/java/com/starrocks/catalog/SchemaTable.java

```diff
@@ -857,7 +873,9 @@ public enum SchemaTableType {
         SCH_BE_COMPACTIONS("BE_COMPACTIONS", "BE_COMPACTIONS", TSchemaTableType.SCH_BE_COMPACTIONS),
         SCH_BE_THREADS("BE_THREADS", "BE_THREADS", TSchemaTableType.SCH_BE_THREADS),
         SCH_BE_LOGS("BE_LOGS", "BE_LOGS", TSchemaTableType.SCH_BE_LOGS),
-        SCH_BE_BVARS("BE_BVARS", "BE_BVARS", TSchemaTableType.SCH_BE_BVARS);
+        SCH_BE_BVARS("BE_BVARS", "BE_BVARS", TSchemaTableType.SCH_BE_BVARS),
+        SCH_BE_CLOUD_NATIVE_COMPACTIONS("BE_CLOUD_NATIVE_COMPACTIONS", "BE_CLOUD_NATIVE_COMPACTIONS",
```

Comment:
if the plan is, be will be split into cn + dn, is this `BE_` naming still ok?

Reply:
> if the plan is, be will be split into cn + dn, is this `BE_` naming still ok?

I'm not sure, the naming convention was restricted by the current system table design.

Reply:
> > if the plan is, be will be split into cn + dn, is this `BE_` naming still ok?
> 
> I'm not sure, the naming convention was restricted by the current system table design.

Some codes rely on the table name to do some special logics.

Reply:
> if the plan is, be will be split into cn + dn, is this `BE_` naming still ok?

@imay How do you think about this?